docs/scos/dev/tutorials-and-howtos/howtos/howto-setup-xdebug-profiling.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "HowTo: Set up XDebug profiling" 
+description: Learn how to setup XDebug profiling in a local development environment.
+template: howto-guide-templatel
+last_updated: Jun 19, 2023
+---
+
+This document explains how to set up *XDebug profiling* in a local development environment to understand in detail the cost of functions and transactions in terms of time and memory demand. XDebug profiling lets you analyze your applications, find out the reason for performance and stability issues, and start profiling your Spryker project.
+
+## Prerequisites
+
+* Working SCOS applicaiton setup with our Docker SDK *.
```

Comment:
What is the meaning of asterisk (*) here?
What does "our" refer to? Spryker or a person that uses the guide?
```suggestion
* Working SCOS application setup with our Docker SDK *.
```

Reply:
- Meant to create bullet point list
- refers to Spryker's Docker SDK as provided via github

Reply:
@ThomasLehnerSpryker I mean the asterisk in the end of the sentence, after "SDK"